src/Views/OperatingSystemView.vala

```diff
@@ -178,6 +178,24 @@ public class About.OperatingSystemView : Gtk.Box {
             child = details_button
         };
 
+        var automatic_updates_switch = new Gtk.Switch () {
+            valign = CENTER
+        };
+
+        var automatic_updates_header = new Granite.HeaderLabel (_("Automatic Updates")) {
```

Comment:
I think `hexpand` would be good here to align the switch to the right like it's done with the action button above

```suggestion
        var automatic_updates_header = new Granite.HeaderLabel (_("Automatic Updates")) {
            hexpand = true,
```